node-commons/src/main/java/org/apache/iotdb/commons/utils/BasicStructureSerDeUtil.java

```diff
@@ -17,18 +17,18 @@
  * under the License.
  */
 
-package org.apache.iotdb.confignode.util;
+package org.apache.iotdb.commons.utils;
 
 import java.nio.ByteBuffer;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
-public class SerializeDeserializeUtil {
+public class BasicStructureSerDeUtil {
```

Comment:
why rename ?    I feel SerializeDeserializeUtil  is better.

Reply:
Mr.Qiao thinks SerDe is better @qiaojialin